docs/manifest.json

```diff
@@ -1763,7 +1763,7 @@
                   },
                   {
                     "title": "Networkless token verification",
-                    "href": "/docs/references/nodejs/token-verification"
+                    "href": "/docs/backend-requests/handling/manual-jwt#networkless-token-verification"
```

Comment:
Hi @alexisintech! I‘m not sure if the URL I added here is the right link for the Node.js/Express section of the sidebar, but the current one 404s 🙇

Reply:
I have a PR to remove the link altogether - please remove this change !